03-task-manager/answers/quizAnswers.txt

```diff
@@ -0,0 +1,17 @@
+1. Quickly explain how middleware is used in an Express application.
+
+Middleware functions can be used to intercept incoming requests and perform various operations on them, such as modifying the request or response objects, validating user input, or handling errors. Middleware can be used at different levels of an Express application, including application-level, router-level, and error-handling middleware, and can help to make applications more secure, reliable, and easier to maintain by customizing the behavior of the request-response cycle.
+
+2. What does CRUD stand for?
+
+CRUD stands for "Create, Read, Update, Delete"
```

Comment:
That's right; the acronym is commonly used because these are the four actions that are most commonly needed when building a REST API application with some sort of persistent backend/database.